hardhat.config.ts

```diff
@@ -32,14 +40,39 @@ const config: HardhatUserConfig = {
       "contracts/HubPool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
     },
   },
+  zksolc: {
+    version: "1.1.0",
+    compilerSource: "docker",
+    settings: {
+      optimizer: {
+        enabled: true,
+      },
+      experimental: {
+        dockerImage: "matterlabs/zksolc",
+        tag: "v1.1.0",
+      },
+    },
+  },
+  zkSyncDeploy: {
+    zkSyncNetwork: "https://zksync2-testnet.zksync.dev",
+    ethNetwork: "goerli", // Can also be the RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
+  },
   networks: {
-    hardhat: { accounts: { accountsBalance: "1000000000000000000000000" } },
+    hardhat: { accounts: { accountsBalance: "1000000000000000000000000" }, zksync: compileZk },
```

Comment:
Does this overwrite the normal artifacts or does it output to a different directory?

Reply:
Outputs to `artifacts-zk` and `cache-zk` because the compiler is diff